lib/trie/trie_test.go

```diff
@@ -435,6 +353,12 @@ func Test_Trie_Hash(t *testing.T) {
 				root: &Node{
 					Key:      []byte{1, 2, 3},
 					SubValue: []byte{1},
+					MerkleValue: []byte{
+						0xa8, 0x13, 0x7c, 0xee, 0xb4, 0xad, 0xea, 0xac,
+						0x9e, 0x5b, 0x37, 0xe2, 0x8e, 0x7d, 0x64, 0x78,
+						0xac, 0xba, 0xb0, 0x6e, 0x90, 0x76, 0xe4, 0x67,
+						0xa1, 0xd8, 0xa2, 0x29, 0x4e, 0x4a, 0xd9, 0xa3,
+					},
```

Comment:
note: merkle values are only calculated and cached if they are absent. In this case, they were absent in the original trie, so they are cached in the final trie.